docs/sources/mimir/release-notes/v2.8.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Grafana Mimir version 2.8 release notes"
+menuTitle: "V2.8 release notes"
+description: "Release notes for Grafana Mimir version 2.8"
+weight: 300
+---
+
+# Grafana Mimir version 2.8 release notes
+
+Grafana Labs is excited to announce version 2.8 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Query-frontend and ruler now use protobuf internal query result payload format by default** This reduces the CPU and memory utilisation of the querier, query-frontend and ruler, as well as reducing network bandwidth consumed between these components.
+- **Query-frontend cached results now contain timestamp** This allows Mimir to check if cached results are still valid based on current TTL configured for tenant. Results cached by previous Mimir version are used until they expire from cache, which can take up to 7 days. If you need to use per-tenant TTL sooner, please flush results cache manually.
+- **Optimized regular expression label matchers** This reduces CPU utilisation in ingesters and store-gateways when running queries containing regular expression label matchers.
+- **Store-gateway now use streaming for LabelNames RPC** This improves memory utilization in store-gateway when calling LabelNames RPC.
+- **Experimental support for using Redis as cache** Mimir now can use Redis for caching results, chunks, index and metadata.
+- **Experimental support for fetching secret from Vault for TLS configuration**.
+- **Experimental support for querying native histograms**. This support is not finalized as the related Prometheus API is also experimental, thus the exact behavior might change in future releases.
```

Comment:
These should be moved to the top; they're experimental, but have the most customer relevance.